rllib/contrib/maddpg/maddpg_policy.py

```diff
@@ -313,11 +308,10 @@ def _build_critic_network(self,
                               act_n,
                               obs_space_n,
                               act_space_n,
-                              hiddens,
-                              activation=None,
```

Comment:
Why this change? It makes the parameters less explicit

Reply:
Yes, it's a simplification. If `hiddens` and `activation` are information that is already given within the config dict (which is also passed into this function), then why pass them in as separate params on top? Also, it's a private helper function that's only called within this class 3 times.

Reply:
Ah, I remember now. The problem is `self.config`, which should not be defined at this point as config is now a super-class ctor argument (for unification reasons: all Policies actors have this arg now) and the super constructor has not been called in this call to _build_critic_network().

Reply:
If we changed it back, we would consequently have to pass in `use_state_preprocessor` as an arg as well. The question would be: Why is it ok to pull some information from self.config, but others has to be passed into this function explicitly? It's not consistent.

Reply:
One feedback I've gotten from users before is that reading directly from config makes it hard to understand what are the parameters of functions (i.e., it's fine to have a config at the top level, but confusing if it's passed whole to lower level modules). It's probably true we pass config dicts around more than necessary and would be better off with python named args / kwargs instead.

Reply:
Ok, makes sense. I'll pass in all three as single params, no `config` anymore. ...

Reply:
Done.